### DISCUSSION TITLE
Conflict with Revit Lookup

### DISCUSSION THREAD

**Discussion by dsquad96** [Nice3point/RevitTemplates#50] [Q&A] [answered]
I just started with the Nice3Point Revit Application Addin template, and when I went to debug (without adding anything to the template), I got hit with an exception around Revit Lookup  (which I already have installed in my revit).
![image](https://github.com/Nice3point/RevitTemplates/assets/10123007/e79017c0-75c7-468c-a479-aa1247e1934c)

Seems there is some conflict with the Revit Lookup Addin? I tried removing Revit Lookup from my addins, and then the template worked perfectly and allowed me to debug normally. Has anyone else encountered this?

**Comment by Nice3point** [comment]
Can you provide the exact RevitLookup version used?

**Comment by dsquad96** [comment]
Sure, it's in Revit 2023 and this is what the "About" window shows:
![image](https://github.com/Nice3point/RevitTemplates/assets/10123007/47001035-d629-4d62-9569-37b1e44220b3)
Thanks for taking a look!

**Reply by Nice3point** [reply, marked as answer]
1.0.0 is not the official version of lookup, apparently you compiled it yourself. And also, seeing on the Ui, outdated. Update the RevitLookup from the official github repository and the problem should be solved https://github.com/jeremytammik/RevitLookup/releases

**Reply by dsquad96** [reply]
Now that you say that, I do remember someone on my team mentioning that they had compiled it themselves. I think they did it so that they could sign the addin so it would stop asking about loading it every time Revit opened (I think). 
So is the issue that when I'm using the Nice3Point template, it's including some newer version of a package and my out of date Revit Lookup is trying to use it?
Thanks!

**Reply by Nice3point** [reply]
yes correct, templates used the latest version, and your lookup used outdated
